step-api-net/KeywordsForTesting/KeywordsForTesting.csproj

```diff
@@ -7,13 +7,15 @@
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.15" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\StepApiFunction\StepApiFunction.csproj" />
```

Comment:
@jeromebrongniart Will this new reference be needed for keyword projects?

Reply:
No: It's only for the KeywordsForTesting project, which is not published